### PR TITLE
:wrench: move install script test from nxxm-src to nxxm releases repository

### PR DESCRIPTION
**Pull request by daminetreg**
Dear @amri04 @Lambourl  

Just a PR to let you notice I moved CI workflows for tests scripts out of nxxm-src and fixed the install folder to use /tmp and not $HOME, as it makes no sense to pollute user's home.

### CODE REVIEW

File: install/install_for_windows.ps1 [hunk 1]
```diff
@@ -1,6 +1,4 @@
-﻿﻿#!/usr/bin/pwsh
-
-$INSTALL_FOLDER="C:\Windows"
+﻿$INSTALL_FOLDER="C:\Windows"
```
Comment (daminetreg) [inline install/install_for_windows.ps1]: @amri04 @Lambourl This is really questionable, isn't that a bit extreme ? 
Installing in his own "Program Files" folder would be really better, and we could use powershell to modify the path to our location : 

```
$oldpath = (Get-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH).path
$newpath = “$oldpath;c:\path\to\folder”
Set-ItemProperty -Path 'Registry::HKEY_LOCAL_MACHINE\System\CurrentControlSet\Control\Session Manager\Environment' -Name PATH -Value $newpath
```

If you don't see any problems regarding this, I'll create a new issue.

Comment (daminetreg) [reply install/install_for_windows.ps1]: A meaning about that @amri04  ? 😆

File: .github/workflows/check_install_script.yaml [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+          if ($env:GITHUB_EVENT_NAME -eq "pull_request") {
+            $script_source = $env:GITHUB_HEAD_REF.replace('refs/heads/', '')
+          } else {
+            $script_source = $env:GITHUB_REF.replace('refs/heads/', '')
+          }
+          Set-ExecutionPolicy Bypass -Scope Process -Force; [System.Net.ServicePointManager]::SecurityProtocol = [System.Net.ServicePointManager]::SecurityProtocol -bor 3072; iex ((New-Object System.Net.WebClient).DownloadString("https://raw.githubusercontent.com/nxxm/nxxm/$script_source/install/install_for_windows.ps1"))
+
+          # Test install succeeeded 
+          if (!(Test-Path "C:\\ProgramData\\nxxm\\nxxm.exe" -PathType leaf)) { exit 1; }
+          if (!(Test-Path "C:\\.nxxm\\0000001\\downloads\\nodejs.zip" -PathType leaf)) { exit 1; }
```
Comment (Lambourl) [inline .github/workflows/check_install_script.yaml]: For the test with gh : We won't be on disk C:\

**Review by Lambourl — APPROVED** [review]

**Review by Lambourl — DISMISSED** [review]

File: install/install_for_windows.ps1 (hunk 1 above)
Comment (daminetreg) [reply install/install_for_windows.ps1]: Ok solved anyway now we install in `C:\ProgramData`

**Pull request merged**